pkg/config/config.go

```diff
@@ -151,7 +151,14 @@ type Config struct {
 
 // Environ returns the config as a list of environment variables.
 func (c *Config) Environ() []string {
-	envs := []string{}
+	ex, err := os.Executable()
+	if err != nil {
+		ex = "soft-serve"
```

Comment:
The default binary name is `soft`
```suggestion
		ex = "soft"
```